### DISCUSSION TITLE
Ikuu \ Tuya TS0011 - No status update with manual operation

### DISCUSSION THREAD

**Discussion by activex** [Koenkk/zigbee2mqtt#8427] [Devices]
Hi all,

I've got a Ikuu \ Tuya TS0011 single gang wired wall switch [Link](https://www.ikuu.com.au/product/single-switch/), registration on Z2M worked fine after the 3rd attempt and I can control the switch through the dashboard and manually.

When I press the switch button manually, the status is not being updated in the dashboard. The status is off, even though all lights are on.

Is that normal behaviour? How can I investigate further or fix this issue.

For context, I use a Conbee II on a raspberry pi 4, running Z2M and homebridge to get my zigbee devices into HomeKit.

Thanks in advance!

**Comment by activex** [comment]
When manually pressing the switch and then trying to refresh the status via the webinterface, I get the following error:

`Error Publish 'get' 'state' to '0x60a423fffe6cb946' failed: 'Error: Read 0x60a423fffe6cb946/1 genOnOff(["onOff"], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'get' 'state' to '0x60a423fffe6cb946' failed: 'Error: Read 0x60a423fffe6cb946/1 genOnOff([\"onOff\"], {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":true,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (no response received)'","meta":{"friendly_name":"0x60a423fffe6cb946"},"type":"zigbee_publish_error"}'
`

**Reply by activex** [reply]
I've removed and rejoined the switch and now I can manually update the status by hitting the refresh button on the dashboard. Triggering the switch manually still does not update the status on the dashboard.

I've also removed the switch and added it to the Ikuu Hub and using the Ikuu app, I was able to connect the switch. Any changes are immediately reflected in the app.

**Reply by Asleep-at-the-wheel** [reply]
I am running into the exact same issue, it's driving me insane.
https://github.com/Koenkk/zigbee2mqtt/issues/8676

Are you able to do some packet sniffing between the hub and light to see how it behaves? I have no access to a hub.

**Reply by activex** [reply]
Wow, you’ve done quite some research on this. I’ll try the config suggestion in the next few days and see if that does anything.
How would I go about sniffing, getting a code dump using the hub?
At the moment I have the hub setup with tuya and use the tuya plugin in homebridge to get it into HomeKit which is working but kind of backwards since it should work just locally through z2m.

**Reply by Asleep-at-the-wheel** [reply]
Never mind, I got to the bottom of it with Koenkks help.

I will get a pull request done and hopefully in the next release this issue should be fixed for Z2MQTT.

**Comment by ievq00** [comment]
I have a similar problem.
There are two ts0011 relays on different zigbee modules.
1 - tyzs3. Status update is working
2 - zs3l. The status is updated only manually via the z2m web interface
![IMG_20210904_232342](https://user-images.githubusercontent.com/90113336/132107432-761e0e49-709d-404d-96c1-98b2d616b155.jpg)
